packages/app/src/app/pages/Sandbox/Editor/Workspace/index.tsx

```diff
@@ -76,7 +76,7 @@ const Workspace = () => {
       )}
 
       {!zenMode && (
-        <div>
```

Comment:
Did this change anything in styling?

Reply:
@MichaelDeBoey Why remove the fragment?

Reply:
@Saeris You didn't remove the fragment, you deleted the `div` and made a fragment of it.